core/controllers/collection_editor.py

```diff
@@ -66,6 +66,7 @@ def get(self, collection_id):
             'SHOW_COLLECTION_NAVIGATION_TAB_STATS': (
                 feconf.SHOW_COLLECTION_NAVIGATION_TAB_STATS),
             'TAG_REGEX': feconf.TAG_REGEX,
+            'title': collection.title,
```

Comment:
@abhaygupta97, I think we are still having some misunderstanding:

You've added an attribute in the get request of  `CollectionEditorPage`, right?

If yes, then I was expecting to have  a test for this attribute inside the `test_editable_collection_handler_get` of `core/controllers/collection_editor_test.py`.

Reply:
I have made changes in get request of `CollectionEditorPage`. But in the test `test_editable_collection_handler_get` request is made to `COLLECTION_EDITOR_DATA_URL_PREFIX` as seen [here](https://github.com/oppia/oppia/blob/develop/core/controllers/collection_editor_test.py#L113) which calls `EditableCollectionDataHandler` as seen [here](https://github.com/oppia/oppia/blob/develop/main.py#L560). But in test I added, request is made to `COLLECTION_EDITOR_URL_PREFIX` which calls `CollectionEditorPage` as seen [here](https://github.com/oppia/oppia/pull/6362/files#diff-07e63f23f436531d7a5239d031d7887bR185). The test is such that after collection title is modified as given [here](https://github.com/oppia/oppia/pull/6362/files#diff-07e63f23f436531d7a5239d031d7887bR45) due to put_json [here](https://github.com/oppia/oppia/pull/6362/files#diff-07e63f23f436531d7a5239d031d7887bR177), it is checked that title is actually changed [here](https://github.com/oppia/oppia/pull/6362/files#diff-07e63f23f436531d7a5239d031d7887bR187).

Reply:
@DubeySandeep @bansalnitish PTAL, thanks.

Reply:
@abhaygupta97, Thanks for explaining I thought that the get request is sending a JSON response.